pytorch_pfn_extras/writing/_tensorboard_writer.py

```diff
@@ -27,10 +27,10 @@ def __init__(
             stats: Optional[KeysView[str]] = None,
             **kwds: Any
     ) -> None:
-        import torch.utils.tensorboard
+        from torch.utils.tensorboard import SummaryWriter
         self._stats = stats
-        self._writer = torch.utils.tensorboard.SummaryWriter(  # type: ignore[no-untyped-call] # NOQA: B950
-            log_dir=out_dir, **kwds)
+        self._writer: Optional[SummaryWriter] = SummaryWriter(
```

Comment:
pytorch_pfn_extras/writing/_tensorboard_writer.py:32: error: Call to untyped function "SummaryWriter" in typed context  [no-untyped-call]